dynatrace-aws-logs.sh

```diff
@@ -91,9 +91,8 @@ arguments:
     if [ -z $2 ] || [[ $2 == "--"* ]]; then echo "Missing value for parameter $1"; print_help_deploy; exit 1; fi
   }
 
-  check_api_token() {
-    URL=$(echo "$TARGET_URL" | sed 's:/*$::')
```

Comment:
This regex was not doing anything useful?

Reply:
It does but I've moved it to 215 line to use url without last "/" in validation functions

Reply:
Oh, I see now. Didn't know the rules of forward referencing in bash